design-system/materials/resets.css

```diff
@@ -0,0 +1,41 @@
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html,
+body {
+  color: var(--font-color-for-text);
+  font-family: var(--font-family);
```

Comment:
I'm going a bit back and forth with this, but I think it kind of makes sense to have this defined here instead of relying on the application/consumer to do so.

Thoughts? @commercetools/shield-team-fe